tests/functional/test_config.py

```diff
@@ -55,6 +55,32 @@ def test_ethereum_network_configs(config, temp_config):
         assert actual.rinkeby.block_time == 15
 
 
+def test_network_gas_limit_default(config):
+    eth_config = config.get_config("ethereum")
+
+    assert eth_config.rinkeby.gas_limit == "auto"
+    assert eth_config.local.gas_limit == "auto"
+
+
+def test_network_gas_limit_config(config, temp_config):
```

Comment:
Maybe a good opportunity to add `@pytest.mark.parametrize("gas_limit", (1234, "1234"))`

Reply:
Agree 🙂 updated in 09d9c975

Reply:
Updated again in 4c502af781eca05c73bfa7299393040f58d2038f to add more test cases